include/podio/tools.h

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+#include <iomanip>
```

Comment:
```suggestion
#include <memory>
```

That should be the only header from the STL that is necessary here, since the others are related to output and should instead be included where they are really needed (if not already the case)